doc/metadata_routing.rst

```diff
@@ -200,14 +214,13 @@ least one metadata. For instance, if an estimator supports ``sample_weight`` in
   in almost all cases the default value when an object is instantiated and
   ensures the user sets the metadata requests explicitly when a metadata is
   passed. The only exception are ``Group*Fold`` splitters.
-- ``"param_name"``: if this estimator is used in a meta-estimator, the
-  meta-estimator should forward ``"param_name"`` as ``sample_weight`` to this
-  estimator. This means the mapping between the metadata required by the
-  object, e.g. ``sample_weight`` and what is provided by the user, e.g.
-  ``my_weights`` is done at the router level, and not by the object, e.g.
-  estimator, itself.
-
-Metadata are requested in the same way for scorers using ``set_score_request``.
+- ``"param_name"``: alias for ``sample_weight`` if we want to pass different weights to
+  different consumers via `fit` and `score`. If aliasing is used in a meta-estimator,
+  the meta-estimator should forward ``"param_name"`` as ``sample_weight`` to the
```

Comment:
aliasing is always mapped and handled in meta-estimator levels, when is it not like that?